### DISCUSSION TITLE
Only a couple of entities get activated

### DISCUSSION THREAD

**Discussion by anderstralala** [bimmerconnected/bimmer_connected#284] [Q&A] [answered]
Hi!
I've installed th integration and only get position and door locks listed as entities. When I run CLI bimmerconnected the list shows a lot more information. I will also submit the fingerprints further on.

![image](https://user-images.githubusercontent.com/84744998/120399373-2a4bd900-c33c-11eb-9558-9ae47cc60a52.png)

![Skärmavbild 2021-06-02 kl  00 49 32](https://user-images.githubusercontent.com/84744998/120399491-6a12c080-c33c-11eb-9596-78881138ee83.png)

**Comment by gerard33** [comment, marked as answer]
The lscType from the fingerprint shows `NOT_SUPPORTED` which means there is not much info from your car available, I think in those cases indeed only the device tracker and lock are.
The BMW integration in Home Assistant only adds entities for which data in the API is available.

From which year is your car? Probably 2015 or 2016?
